widgets/src/components/common/form/SchemaEditor/JsonMultiFieldContainer.js

```diff
@@ -43,7 +54,7 @@ class JsonMultiFieldContainer extends React.Component {
 
   handleClickAddSchema(formSchemaId, name, schema) {
     const { schemas, onChange } = this.props;
-    const uiSchemas = JSON.parse(schemas);
+    const uiSchemas = JsonMultiFieldContainer.parseSchema(schemas);
```

Comment:
can't we use the `this` keyword here instead of using the class name?

Reply:
As this function does not use component's `this` inside of it, it has to be a static function. Also, eslint would complain that class method is expected to use `this`.